libs/watt/src/lib/configuration/watt-date-adapter.ts

```diff
@@ -14,22 +14,14 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { Inject, Optional } from '@angular/core';
-import { DateFnsAdapter } from '@angular/material-date-fns-adapter';
-import { MAT_DATE_LOCALE } from '@angular/material/core';
-import { da as daLocale, enGB as enGBLocale } from 'date-fns/locale';
+import { NativeDateAdapter } from '@angular/material/core';
 
 export type WattSupportedLocales = 'da' | 'en';
```

Comment:
Question: wouldn't it be better to actually use the real locale name `en-GB`? I know that we currently only support Danish and English but because we're doing it like this we need checks like `language === danishLocale ? danishLocale`

Reply:
@dzhavat did a the research, it should be okay :-)

Reply:
We can talk about this in the follow-up refactoring issue. I resolve for now :)

Reply:
@frontend-specialisten you can see some details about this in the first resolved comment in this PR. :)